pkg/client/request.go

```diff
@@ -483,7 +483,8 @@ func (r *Request) Do() Result {
 			continue
 		}
 
-		if resp.StatusCode == http.StatusServiceUnavailable {
+		// Check to see if we got a 429 Too Many Requests response code.
+		if resp.StatusCode == 429 {
```

Comment:
Yes, I wondered about a constant but I was not sure where we should put it. I'll add it to either apiserver.go or errors.go. Thank you.